### DISCUSSION TITLE
Installation of EDDY Bigtreetech USB V1 on K1x

### DISCUSSION THREAD

**Discussion by jaxxtheone** [Guilouz/Creality-Helper-Script-Wiki#602] [Features Request]
SOLVED bellow by @BallaSwag

Guilouz, will you consider adding Eddy USB V1 on Helper-Script? Most of us think that any mesh 3x3, 4x4 or 5x5 to say, normal people, spend on a KAMP mesh may be greatly reduced? It doesn't seem so complicated when you use normal Klipper but I don't know about Linux used on our mainboards. It should not be that difficult, I tracked  [https://makerworld.com/en/models/480049#profileId-391522](url) as the adapter and [https://github.com/bigtreetech/Eddy?tab=readme-ov-file#btt-eddy-dimensions-and-probe-location](url) as the github.

**Comment by jaxxtheone** [comment]
Complementing my request I found today []https://github.com/ballaswag/creality_k1_klipper_mod(url) that has the path to implementing Eddy.

**Reply by Futz3r** [reply]
I tried twice to install the one from Ballaswag. every time klipper hangs up and gets no feedback from the MCU.

**Comment by jaxxtheone** [comment]
@BallaSwagg thanks for the SOLVE!

Newest guppy mod comes with a flasher and prebuilt eddy firmware, so you can do it all from the k1.
Full guide: [https://ballaswag.github.io/blog/creality-k1-btt-eddy-guide/](https://ballaswag.github.io/blog/creality-k1-btt-eddy-guide/?fbclid=IwZXh0bgNhZW0CMTAAAR0MrG67oHlQJtUyeCmzsw5bIH0eF3DhOGotcVzS7Dx-QWQPzDQeXExZeqE_aem_Cnb3nxRroIFEnjDNxHx7dw)

**Reply by roofuskit** [reply]
Going this path will complete disable PR Touch though. It would be nice to have a method that does not destroy your ability to us PR touch for an Auto z-offset.

**Reply by FelixOstermeier** [reply]
Has anyone tried flashing the eddy firmware on e.g. a RPi and then using it on the stock creality K1x Klipper? Might give it a shot at some point...
